src/Avalonia.Controls/Primitives/UniformGrid.cs

```diff
@@ -126,6 +172,10 @@ private void UpdateRowsAndColumns()
                 SetCurrentValue(FirstColumnProperty, 0);
             }
 
+            // return if no shuold calculate
```

Comment:
Comment is a big ambiguous in my opinion. Also `should` misspelled

Reply:
@EricZeiberg I apologize for the comment, English is not my native language, anyway I think it is clear what this condition does.

Reply:
Tbh I'm not sure why this `if` statement is more deserving of a comment than any of the other lines in this method, but if you deem it necessary;

```suggestion
            // Return if no calculation required
```